### PR TITLE
[TW-188] Update Postman web and desktop terms to sentence case

### PR DESCRIPTION
**Pull request by MatthewPinsker-postman**

### CODE REVIEW

File: src/pages/docs/administration/managing-your-team/managing-your-team.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Postman provides a variety of options to customize your team's experience, from
 
 Once you've created your Postman team, you can take the following steps to enable an efficient onboarding process for end users:
 
-* Use the [Onboarding checklist](/docs/administration/onboarding-checklist/) to ensure that both the Postman Desktop App and Postman for Web are accessible on employee devices and function as expected.
+* Use the [Onboarding checklist](/docs/administration/onboarding-checklist/) to ensure that Postman is accessible on employee devices and function as expected.
```
Comment (petewitcher) [inline src/pages/docs/administration/managing-your-team/managing-your-team.md:48]: I think 'function' should be 'functions'.

**Review by petewitcher — APPROVED** [review]
LGTM with one small change

**Review by mark-anthony-postman — APPROVED** [review]
LGTM! 🌟

File: src/pages/docs/administration/managing-your-team/managing-your-team.md (hunk 1 above)
Comment (MatthewPinsker-postman) [reply src/pages/docs/administration/managing-your-team/managing-your-team.md]: Yes, I agree. I just pushed the change to the branch. Good catch!

**Review by jkonrath-postman — APPROVED** [review]
LGTM!

**Pull request merged**